dask/highlevelgraph.py

```diff
@@ -404,7 +404,9 @@ def __dask_distributed_pack__(
             dependencies = {k: v - alias_keys for k, v in dependencies.items()}
         # - Add in deps for any missing keys
         missing_keys = dsk.keys() - dependencies.keys()
-        dependencies.update(
+
+        # TODO: add some overloads to keys_in_task to get the return types right.
+        dependencies.update(  # type: ignore
```

Comment:
I guess mypy was saying "you can't update a read only mapping". mypy is absolutely right with that deepcopy.
```suggestion
        dependencies.update(
```